packages/mobile_app/lib/common/ambiance/ambiance_subject_settings.dart

```diff
@@ -0,0 +1,175 @@
+import 'common.dart';
+
+class AmbianceSubjectChange extends StatefulWidget {
```

Comment:
Any chance sharing more code between `AmbianceSubjectChange` and `AmbianceSubjectNew` (additional validation for example)?

Reply:
I'll see if it worth it